### PR TITLE
update to 5.32.1

### PR DESCRIPTION
**Pull request by katietz**
# Perl v5.32.1

- Update recipe from forge
- Fix builds for OSX and Windows
- Unblocks `perl-xml-parser-feedstock`

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
We might need to reset the `build_number`
https://github.com/AnacondaRecipes/perl-feedstock/blob/62ac3b23452a0fac722b2673f4f7e546689f86d2/recipe/meta.yaml#L22

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,8 +1,11 @@
-robocopy %SRC_DIR%\perl\ %LIBRARY_PREFIX%\ *.* /E
-if %ERRORLEVEL% GEQ 8 exit 1
-
-REM There's a bat file in here that says it is better than exe.
-REM Let's trust the strawberry perl folks.
-del %LIBRARY_BIN%\perlglob.exe
-
-copy %SRC_DIR%\licenses\perl %LIBRARY_PREFIX%\perl_licenses
+bash -xeu %RECIPE_DIR%\replace-strawberry-perl-paths.sh
```
Comment (ryanskeith) [inline recipe/bld.bat:1]: I did not know bash was a command we could use in our windows builder.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,37 +1,99 @@
...
-  # sha1 hashes published at: http://strawberryperl.com/releases.html
-  sha1: 7996609176451c54864f700ffa299ee81712d8bd                                                                  # [win64]
-  sha1: 73564a7cef397c4e0da93612c00fa5f482e652b3                                                                  # [win32]
+  - url: https://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.xz                                                 # [unix]
+    sha256: 57cc47c735c8300a8ce2fa0643507b44c4ae59012bfdad0121313db639e02309                                        # [unix]
+    patches:                                                                                                        # [unix]
+      - 0001-Use-userelocatableinc-together-with-useshrplib.patch                                                   # [unix]
+  - url: https://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
+    url: https://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
```
Comment (ryanskeith) [inline recipe/meta.yaml:16]: Do we need to keep around the win32 hashes anymore?

**Review by ryanskeith — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (ELundby45) [reply recipe/meta.yaml]: We shouldn't. I went ahead and removed those. Thanks!

**Review by ELundby45 — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/conda_build_config.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+iMACOSX_SDK_VERSION:  # [osx and x86_64]
```
Comment (cbouss) [inline recipe/conda_build_config.yaml:1]: iMACOSX_SDK_VERSION : the i at the beginning looks like a typo. But this doesn't look to be used either.

**Review by cbouss — APPROVED** [review]
Apart from this seems fine to me.

File: recipe/conda_build_config.yaml (hunk 3 above)
Comment (ELundby45) [reply recipe/conda_build_config.yaml]: Nice catch. Thanks!

**Pull request merged**